### ISSUE TITLE
Allow sentry-wizard to be used in expo projects

### ISSUE THREAD

**Issue by grossadamm** [getsentry/sentry-wizard#192]
Good day!

TL;DR: Would it be possible to add support for passing in the sentry auth token as an environment variable?

**Context**

When building using eas-cli there is an option to have a `postInstall` build hook which runs after expo has done it's pre-install step. Pre-install essentially means creating a close to normal react-native project with separate `ios` and `android` folders.

```
 "scripts": {
    "eas-build-on-success": "echo 123",
    "eas-build-post-install": "./postInstallHook",
```

Right now, in my `postInstallHook` I execute sentry-wizard, but there are two things that don't work out of the box when running this as part of CI:

1) Support for passing in the sentry auth token as an environment variable
2) patching the `android/app/build.gradle`

I'm not blocked since these steps can be done manually as follows:

```
#!/bin/bash

npx @sentry/wizard --quiet --skip-connect -i reactNative -p ios android
node alternativeToEnvsubst.js > sentry.properties  # replace $SENTRY_AUTH_TOKEN variable in sentry.properties.template

if [ -f "android/app/build.gradle" ]; then
  cp sentry.properties android/sentry.properties
  sed '/apply from: new File(reactNativeRoot, "react.gradle")/a\
apply from: new File(["node", "--print", "require.resolve('"'"'@sentry/react-native/package.json'"'"')"].execute().text.trim(), "../sentry.gradle")' android/app/build.gradle > android/app/build.gradle.tmp
  mv android/app/build.gradle.tmp android/app/build.gradle
fi
if [ -f "ios/Podfile" ]; then
  cp sentry.properties ios/sentry.properties
  cd ios && pod install && cd ..
fi
```

`alternativeToEnvsubst.js` looks for an env var `defaults.auth.token=$SENTRY_AUTH_TOKEN` and fills in the value.

Thank you very much for providing this tool and such an awesome service!
